cli/add_domain.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Strip the subdomain of whitespace
+DOMAIN=$(echo "$1" | xargs)
+
+# "who am i" responds correctly even with sudo
+NEST_USER=$(who am i | awk '{print $1}')
+
+# We should validate the domain.
+FQDN_REGEX='^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$'
+
+if [[ ! $DOMAIN =~ $FQDN_REGEX ]]; then
+    echo "Invalid FQDN"
+    exit 1
+fi
+
+if [ "$NEST_USER" = "root" ]; then
+	echo "Command cannot be run as root!"
+	exit 1
+fi
+
+NEEDED_CNAME="$USER.hackclub.app"
```

Comment:
```suggestion
NEEDED_CNAME="$USER.hackclub.app."
```
From my testing, the `dig` command will return the full DNS label with a trailing `.`